server/core/src/https/ui_html.html

```diff
@@ -5,35 +5,41 @@
         <meta charset="utf-8" />
         <meta name="theme-color" content="white" />
         <meta name="viewport" content="width=device-width" />
-        <title>{}</title>
-
-        <link rel="icon" href="/pkg/img/favicon.png" />
+        <title>{display_name}</title>
         <link rel="manifest" href="/manifest.webmanifest" />
-        <link rel="apple-touch-icon" href="/pkg/img/logo-256.png" />
+
+        <link rel="icon"
+            href="/pkg/img/favicon.png?v={cache_buster_key}" />
```

Comment:
Doesn't this cause the client to not cache *anything* at all? 

A cleaner option could just be to have /pkg/{pkg version hash}/img/.... that way the resource dir is stable, and we don't need cache busting at all, and when we change versions, the resource path changes causing the client to refetch the item. Also pretty easy to achieve with route matching.

Reply:
No it'll cache based on the URL, pretty common way of doing it that we use elsewhere already for JavaScript - but we use the file hash for those since we are already calculating it

Reply:
It's the "don't have to make magic routers to update versions" equivalent